storage/storage-samples/storage-rv-sample/src/main/java/org/fido/iot/storage/To0DbStorage.java

```diff
@@ -98,6 +100,100 @@ public long storeRedirectBlob(Composite voucher, long requestedWait, byte[] sign
     throw new ResourceNotFoundException(guid.toString());
   }
 
+  @Override
+  public List<String> getOvKeysAllowlist() {
+
+    List<String> keysInAllowList = new ArrayList<>();
+
+    String sql = "SELECT PUBLIC_KEY_HASH FROM OV_KEYS_ALLOWLIST";
+
+    try (Connection conn = dataSource.getConnection();
+        PreparedStatement pstmt = conn.prepareStatement(sql)) {
+
+      try (ResultSet rs = pstmt.executeQuery()) {
+        while (rs.next()) {
+          keysInAllowList.add(rs.getString(1));
+        }
+      }
+
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
+    return keysInAllowList;
+  }
+
+  @Override
+  public void setOvKeysAllowlist() {
+
+    List<String> allowlist = new ArrayList<>();
```

Comment:
some comment about what these default keys represent is needed here

Reply:
Implementation changed, the hash values are now being read from a properties file with appropriate key name.